recipes/fontconfig/meson/conanfile.py

```diff
@@ -3,16 +3,15 @@
 from conan.tools.env import VirtualBuildEnv
 from conan.tools.files import (
     apply_conandata_patches, copy, export_conandata_patches, get,
-    replace_in_file, rm, rmdir
+    rm, rmdir
 )
 from conan.tools.gnu import PkgConfigDeps
 from conan.tools.layout import basic_layout
 from conan.tools.meson import Meson, MesonToolchain
-from conan.tools.scm import Version
 
 import os
 
-required_conan_version = ">=1.53.0"
+required_conan_version = ">=1.64.0 <2 || >=2.2.0"
```

Comment:
@jwillikers Why was this massive bump to the required Conan version necessary?

Related to https://github.com/conan-io/conan/issues/16140